bindings/azure/blobstorage/blobstorage.go

```diff
@@ -28,6 +29,7 @@ const (
 	contentLanguage    = "ContentLanguage"
 	contentDisposition = "ContentDisposition"
 	cacheControl       = "CacheControl"
+	getBlobRetryCount  = 10
```

Comment:
I would rename from getBlobRetryCount to defaultGetblobRetryCount. 

wouldn't it make this as metadata to configurable ?

Reply:
Sure, we can do that.

Reply:
Check latest commit.